homeassistant/components/media_player/squeezebox.py

```diff
@@ -60,129 +66,111 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                       host, port, error)
         return False
 
-    # Combine it with port to allow multiple servers at the same host
-    key = "{}:{}".format(ipaddr, port)
```

Comment:
I'll revert back to the old behaviour, but we will need to change the discovery method of Squeezebox. It doesn't detect the port, causing #4222.